MMM-Spotify.css

```diff
@@ -57,16 +61,16 @@
   justify-content: flex-start;
 }
 
-#SPOTIFY_INFO .infoicon {
-  width: 20px;
-  margin-right:10px;
+.infoicon {
+  font-size: 24px;
```

Comment:
it broke icon size on other interfaces
I delete it and it's correct

Reply:
Ill do it for just the minimalist mode.